lib/message_builder.rb

```diff
@@ -55,38 +47,56 @@ def comments(pull_request)
     " comments"
   end
 
-  def these
-    return "This" if @old_pull_requests == "1"
-    "These"
+  def these(items)
+    if items == 1
+      'This'
+    else
+      'These'
+    end
   end
 
-  def pr_plural
-    return "pull request has" if @old_pull_requests == "1"
-    "pull requests have"
+  def pr_plural(prs)
+    if prs == 1
+      'pull request has'
+    else
+      'pull requests have'
+    end
   end
 
   def present(pull_request, index)
     pr = pull_requests[pull_request]
+    days = age_in_days(pr)
     <<-EOF.gsub(/^\s+/, '')
-    #{index}\) *#{pr["repo"]}* | #{pr["author"]}
+    #{index}\) *#{pr["repo"]}* | #{pr["author"]} (last updated #{days} #{days_plural(days)} ago)
     <#{pr["link"]}|#{pr["title"]}> - #{pr["comments_count"]}#{comments(pull_request)}
     EOF
   end
 
+  def age_in_days(pull_request)
+    (Date.today - pull_request['updated']).to_i
+  end
+
+  def days_plural(days)
+    if days == 1
+      'day'
+    else
+      'days'
+    end
+  end
+
   def informative
-   if @pull_requests == {}
+    if @pull_requests.empty?
       no_pull_requests
     else
       list_pull_requests
-      @report
     end
   end
 
   def angry
-    @alert = ""
-    check_old_pull_requests
-    return @alert if @old_pull_requests.length > 0
-    ""
+    if old_pull_requests.empty?
+      ":clap: How can I be angry when you've closed all of your PRs? Well done! Have a :cookie:! :clap:"
```

Comment:
@binaryberry Agreed, the Angry Seal should be an alarm (we're a long way from that here, alas). I've rebased this branch and squashed that commit.